test/unit/components/testStringUtils.js

```diff
@@ -3,15 +3,308 @@
 const expect = require('chai').expect;
 const prefixHttps = require('app/components/string-utils').prefixHttps;
 
-describe('prefixHttps()', () => {
 
-    it('should add HTTPS to the URL if missing', (done) => {
-        expect(prefixHttps('https://test.com')).to.equal('https://test.com');
-        expect(prefixHttps('http://test.com')).to.equal('http://test.com');
-        expect(prefixHttps('test.com')).to.equal('https://test.com');
-        expect(prefixHttps('    test.com')).to.equal('https://test.com');
-        expect(prefixHttps('    test      .        com')).to.equal('https://test.com');
-        done();
+describe('prefixHttps', () => {
```

Comment:
We have many positive scenarios not many negatives. What would be the outcome of www.myfakeSite/platform.hmcts.net for example?

Reply:
So this scenario is not covered then, anyone could with a little coding knowledge can bypass these URLs checks

Reply:
This is now changed to check against hostname